server/src/test/java/org/opensearch/cluster/service/ClusterManagerTaskThrottlerTests.java

```diff
@@ -45,6 +45,7 @@ public class ClusterManagerTaskThrottlerTests extends OpenSearchTestCase {
     private DiscoveryNode localNode;
     private DiscoveryNode[] allNodes;
     private ClusterManagerThrottlingStats throttlingStats;
+    private Settings settings = Settings.EMPTY;
```

Comment:
Since we always use Settings.EMPTY except when otherwise there's really no need for this local test class state. I would remove this (and possibly other) `private` variables. It just adds more confusion, especially here.

Reply:
Ack, removed local class state.